server/index.js

```diff
@@ -78,7 +78,7 @@ app.get('/metrics', (req, res, next) => {
   // empty bearer token.
   if (expectedToken && segments.length === 2 && segments[1] === expectedToken) {
     res.set('Content-Type', prometheus.register.contentType)
-    res.end(prometheus.register.metrics())
+    res.end(await prometheus.register.metrics())
```

Comment:
The changes happening here are identical to [what happened in](https://github.com/algorithmiaio/web/pull/2552) web recently.